usecases/steg_asymptotic_2sensors/test_optimize.py

```diff
@@ -0,0 +1,129 @@
+        # assert(b == data['b'])
+        # assert(c == data['c'])
+        # assert(d == data['d'])
+        # assert(e == data['e'])
+        single_model_error = AsymptoticModelError(x_Mat_sensors, data_MatB, data_MatC, data_Mat_B_C)
+            # read in the parameters that are given in the experiment (in this case constant offset a)
+        parameter = single_model_error.get_parameter_dict()
+            # add parameters that are model parameters (not given in the experimental data)
+        parameter.define("mA")
+        parameter.define("mB") #! was b
+        self.add(single_model_error, parameter)
+
+        # define one shared latent parameter (only one parameter for all models)
+        self.latent.add_by_name('mA')
+#        self.latent.add_by_name('mB')
+
+
+class TestOptimize(unittest.TestCase):
+    def test_asymptotic_virtual_data(self):
+        virtual_experiment_metadata_file = \
+            Path(__file__).parent / 'virtual_experiment_asymptotic_model_meta.yaml'
+        with open(virtual_experiment_metadata_file, "r") as f:
+            d = yaml.load(f, Loader=yaml.FullLoader)
+
+        all_experiments_model_error = \
+            MultiAsymptoticModelError('virtual_experiment_asymptotic_model_meta.yaml',
+                                  'virtual_experiment_asymptotic_model_data.yaml')
+
+        start_vector = np.array([0.7])
+        result = least_squares(all_experiments_model_error, start_vector)
+
+        # assert quadratic component of virtual experiment to be zero
+        self.assertAlmostEqual(d['mB'], 0.)
+        # check to obtain exact asymptotic coefficient
+        self.assertAlmostEqual(result.x[0], d['mA'], 5)
+        # check cost function to be zero
+        self.assertAlmostEqual(result.cost, 0)
+
+    def test_asymptotic_virtual_data_split(self):
+        virtual_experiment_metadata_file = \
+            Path(__file__).parent / 'virtual_experiment_asymptotic_model_with_noise_meta.yaml'
+        with open(virtual_experiment_metadata_file, "r") as f:
+            d = yaml.load(f, Loader=yaml.FullLoader)
+
+        all_experiments_model_error = \
+            MultiAsymptoticModelError('virtual_experiment_asymptotic_model_with_noise_meta.yaml',
+                                  'virtual_experiment_asymptotic_model_with_noise_data.yaml')
+
+        def weighted_all_experiments_model_error(prm_vector):
+            all_model_errors = all_experiments_model_error.evaluate(prm_vector)
+            obj = 0.
+            for key, me in all_model_errors.items():
+                # obj = obj + np.dot(me['MatB'], me['MatB']) / d['sigma_noise_Mat'] ** 2 + \
+               #       np.dot(me['MatC'], me['MatC']) / d['sigma_noise_Mat'] ** 2 \
+                #         +  np.dot(me['Mat_B_C'], me['Mat_B_C']) / d['sigma_noise_Mat'] ** 2
+                obj = obj + np.dot(me['Mat_B_C'], me['Mat_B_C']) / d['sigma_noise_Mat'] ** 2 
+            return np.sqrt(obj)
+
+        weighted_start_vector = np.array([0.7])

```

Comment:
did you check that the start vector is somehow close to your exact values? Since this is a local iteration that should match. For extending this to a Bayesian procedure, we need priors that similar reflect prior knowledge about our parameters.